packages/@guardian/source-foundations/src/accessibility/accessibility.test.ts

```diff
@@ -0,0 +1,45 @@
+import { transform } from 'lightningcss';
+import { focusHalo, focusHaloSpaced } from './focus-halo';
+import { visuallyHidden } from './visually-hidden';
+
+it('should provide valid CSS for focusHalo', () => {
```

Comment:
a bit nit picky, but how do you feel about something like this? (it took me a sec to understand the intention of the test)
```suggestion
test('focusHalo is valid CSS', () => {
```

Reply:
agreed 👍

Reply:
llovely thanks